packages/@storefront/flux-capacitor/src/core/reducers/data/site-params.ts

```diff
@@ -0,0 +1,17 @@
+import Actions from '../../actions';
+import Store from '../../store';
+
+export type Action = Actions.ReceiveSiteParams;
+export type State = Store.SiteParams[];
+
+export const DEFAULT_PARAMS = <any>[];
```

Comment:
Does this need a type cast? I think an empty array still conforms to the type (maybe you have to annotate the variable with a type).